frontend/src/pages/team/Devices/dialogs/DeviceCredentialsDialog.vue

```diff
@@ -56,6 +57,9 @@ export default {
             device: null
         }
     },
+    async mounted () {
+        this.base_url = (await settings.getSettings()).base_url
```

Comment:
The platform settings are preloaded in the Vue store, so you don't need to get them again.

You can use `mapState` to add `settings` as a local computed property, and then you should be able to just use `this.settings.base_url`.

See https://github.com/flowforge/flowforge/blob/main/frontend/src/components/auth/Credentials.vue#L28 for a mapState example that adds `settings`.

Reply:
Updated, (Not 100% sure why it needs to be `mapState('account',['settings')` e.g. why it needs the `account` at the start but it works....